package.json

```diff
@@ -77,7 +77,7 @@
     "@hmpps-book-secure-move-frameworks/1.3.0": "git+https://github.com/ministryofjustice/hmpps-book-secure-move-frameworks.git#v1.3.0",
     "@hmpps-book-secure-move-frameworks/1.4.0": "git+https://github.com/ministryofjustice/hmpps-book-secure-move-frameworks.git#v1.4.0",
     "@ministryofjustice/frontend": "0.0.19-alpha",
-    "@promster/express": "4.1.12",
+    "@promster/express": "4.1.14",
```

Comment:
Could now `fixup` this change into the commit that introduces it within this PR?

Reply:
sure